articles/introducing-fleet-ultimate.md

```diff
@@ -0,0 +1,44 @@
+# Introducing CIS benchmarks, managed-cloud hosting and custom calculator in the new Fleet Ultimate plan.
+
+![Fleet ULTIMATE](../website/assets/images/articles/happy-1st-anniversary-fleet-cover-800x450@2x.png)
```

Comment:
@jarodreyes Do we have a hero image for this article? 
No worries if not, I'm just making sure that the reused image from https://fleetdm.com/announcements/happy-1st-anniversary-fleet is intended.